app/Helpers/SshMultiplexingHelper.php

```
@@ -87,15 +87,27 @@ public static function establishNewMultiplexedConnection(Server $server): bool
-        self::clearConnectionMetadata($server);
+        $checkProcess = Process::run(self::muxControlCommand($server, 'check'));
+        $stopProcess = Process::run(self::muxControlCommand($server, 'stop'));
+
+        if ($stopProcess->successful() && preg_match('/pid=(\d+)/', $checkProcess->output().$checkProcess->errorOutput(), $matches)) {
+            self::markMuxProcessAsRetiring($matches[1]);
+        }
+    }
+
+    public static function markMuxProcessAsRetiring(string $pid): void
+    {
+        Cache::forever(self::muxProcessRetirementKey($pid), true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'markMuxProcessAsRetiring'. (undefined)

(StaticAccess)